scripts/create-usns.py

```diff
@@ -123,7 +123,7 @@ def guess_binary_links(binary, version, sources):
         # Build endpoint
         endpoint = notice_endpoint
         if http_method == "PUT":
-            endpoint = f"{notice_endpoint}/{notice['id']}"
+            endpoint = f"{notice_endpoint}/USN-{notice['id']}"
```

Comment:
I thought the `USN` would be part of the ID. But we can discuss this down the line, no need to block this PR.

Reply:
It is, but this script is used to import data from the `database.json` file, and all the ID's in there are not prefixed.

Reply:
Oooooooh, I see. Great :+1: